src/views/Pools/index.tsx

```diff
@@ -268,7 +268,7 @@ const Pools: React.FC = () => {
                 {t('Search')}
               </Text>
               <ControlStretch>
-                <SearchInput onChange={handleChangeSearchQuery} placeholder="Search pools" />
+                <SearchInput onChange={handleChangeSearchQuery} placeholder={t('Search Pools')} />
```

Comment:
This is actually translated inside SearchInput. There is `t(placeholder)` in there. I'm not sure which way is better though 😄 
If you want you can refactor SearchInput to accept already translated string + also change SearchInput in Farms.tsx. But I think it's not necessary.

Reply:
I removed translation only the translation key is fixed